### PR TITLE
#266 Apply header and menu

### PR DESCRIPTION
**Pull request by ExiRain**
- Updated package to v0.0.27
- Added English translation support for the application

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ This repo will primarily contain:
 - Install Python
 - Navigate to setup and run `python3 generate-certs.py`
 - Navigate to current repo and run `docker-compose up -d`
+- For GUI make sure that Dockerfile.dev have header package include with version specified in package.json `COPY ./exirain-header-0.0.27.tgz ./`
```
Comment (1AhmedYasser) [inline README.md:32]: Why do you need to include it in dockerfile.dev ? , and why not just add it to dockerfile.dev ?

**Review by 1AhmedYasser — CHANGES_REQUESTED** [review]

File: GUI/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "prettier": "prettier --write \"{,!(node_modules)/**/}*.{ts,tsx,js,json,css,less,scss}\""
   },
   "dependencies": {
-    "@exirain/header": "^0.0.26",
+    "@exirain/header": "file:exirain-header-0.0.27.tgz",
```
[the remark this answers is not in the document]
Comment (ExiRain) [reply GUI/package.json:14]: File is used for consistency reason, previously i was using my personal NPM account to store package, but if some one else gonna work with this task then they wont have access to it and therefore its better to keep 1 way of using package within project until more suitable solution would be established.

File: README.md (hunk 1 above)
Comment (ExiRain) [reply README.md:32]: Yeh i will update Dockerfile.dev but this is case if some one will miss this part out.

**Comment by 1AhmedYasser** [comment]
@turnerrainer @PaulaMerle The same approach is currently used in analytics & service module as well, will approve the current approach for the time being, but its needs to be changed to use an `NPM` package across all modules

**Review by 1AhmedYasser — APPROVED** [review]

**Pull request merged**